pkg/network/public/public.go

```diff
@@ -26,6 +26,8 @@ import (
 const (
 	toZosVeth                   = "tozos" // veth pair from br-pub to zos
 	publicNsMACDerivationSuffix = "-public"
+	testMacvlan                 = "pubtestmacvlan"
```

Comment:
weird name, this can just be `pub` or something. it's inside a namespace anyway so no chances of name collision.